home/models.py

```diff
@@ -404,6 +404,7 @@ class Status(TextChoices):
         VERIFIED = "verified"
         PENDING = "pending"
         REJECTED = "rejected"
+        DELETED = "deleted"
```

Comment:
Don't think this status should exist. Why would any review in the database have a status of "deleted"? Wouldn't any review with this status just be deleted from the db instead?

Reply:
Yes, but I needed a way to mark the review for deletion so I could skip all the validation stuff in /admin. There won't ever be a review in the DB with a status of `deleted`, I just thought this was better than just using strings.

Reply:
It's not better than using a string because including it in the model implies we are allowing reviews with this status in the db. It doesn't make sense to have an enum value that can never be held by a model.